spec/helpers/spotlight/pages_helper_spec.rb

```diff
@@ -82,12 +82,12 @@ module Spotlight
       end
       describe 'nestable data attributes hash' do
         it 'returns the appropriate hash for feature pages' do
-          expect(helper.nestable_data_attributes_hash('feature_pages')).to eq "data-max-depth": '2',
-                                                                              "data-expand-btn-HTML": '',
-                                                                              "data-collapse-btn-HTML": ''
+          expect(helper.nestable_data_attributes_hash('feature_pages')).to eq("data-max-depth" => '2',
+                                                                              "data-expand-btn-HTML" => '',
+                                                                              "data-collapse-btn-HTML" => '')
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.